Tests/scripts/collect_tests/collect_tests.py

```diff
@@ -579,7 +579,7 @@ def _get_changed_files(self) -> tuple[str, ...]:
 
         # diff is formatted as `M  foo.json\n A  bar.py\n ...`, turning it into ('foo.json', 'bar.py', ...).
         for line in diff.splitlines():
-            match len(parts := line.split()):
+            match len(parts := line.split('\t')):
```

Comment:
```suggestion
            match len(parts := re.split(r'\s', line):
```
we can use regex here

Reply:
i rather keep it readable. 
`\s` is invalid, see the ticket this issue fixes - caused by files with whitespaces in their name.

Reply:
\t will not work for renamed files.
Regarding the original problem, you can use the max line argument.

Reply:
following offline discussion, `\t` it is